### DISCUSSION TITLE
BAD TERMINATION

### DISCUSSION THREAD

**Discussion by Richard-happy** [idaholab/moose#23836] [Q&A General] [answered]
Hello,when I ran my input file, it was OK using 20*20*20 mesh. However, it seemed someting went wrong when using 50*50*50 mesh.Here is the report.

```
===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 13193 RUNNING AT ml64
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Killed (signal 9)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
```

And my input file is appended.
[nonlinear_TE.txt](https://github.com/idaholab/moose/files/11062783/nonlinear_TE.txt)

**Comment by cticenhour** [comment, marked as answer]
Using your input, a simulation with a 50x50x50 mesh contains quite a large number of nonlinear DoFs (530,000+) and over 1.5 million auxiliary DoFs. My first guess would be that you ran out of memory on your machine.  I ran this input over 10 cores on a laptop with 64GB of RAM, and it was quickly filled up.

My suggestion is to coarsen the mesh to something less than 50x50x50. For a simple input like you've shown here, you should be able to get by with something much less fine. If you end up deciding that refinement is necessary as your input file becomes more complex, look into mesh adaptivity or running this on a higher memory machine (like a cluster or HPC system).

**Reply by Richard-happy** [reply]
OK, thank you for your suggestion, cticenhour
